terraform/layers/app/iam.tf

```diff
@@ -0,0 +1,50 @@
+// Find base module
+module "base" {
+  source = "../base"
+}
+
+output "s3_name" {
+  value = module.base.s3_webfiles_arn
+}
+
+// Create IAM role
+resource "aws_iam_role" "s3_webfiles_access" {
+  name = "s3_webfiles_access"
```

Comment:
need to use the naming convention here, the role will be visible within the console so we need to know what env/project it applies to